api/v1alpha1/jobset_types.go

```diff
@@ -130,6 +130,11 @@ type Network struct {
 	// <jobSet.name>-<spec.replicatedJob.name>-<job-index>-<pod-index>.<jobSet.name>-<spec.replicatedJob.name>
```

Comment:
Can you update this comment on `EnableDNSHostnames` since now the subdomain name format will not necessarily be `<jobSet.name>-<spec.replicatedJob.name>` if the user specifies it.